flytekit/types/file/__init__.py

```diff
@@ -1 +1,54 @@
+import typing
+
 from .file import FlyteFile
+
+# The following section provides some predefined aliases for commonly used FlyteFile formats.
+# This makes their usage extremely simple for the users. Please keep the list sorted.
+
+
+HDF5EncodedFile = FlyteFile[typing.TypeVar("hdf5")]
```

Comment:
I still don't like this usage of TypeVar, it feels wrong, but we can always change it in the future without breaking people.

Reply:
ya, it is required today for linter